stanford/bin/send-email.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+from email.mime.text import MIMEText
+from subprocess import call
+import sys
+
+def send(recipient, sender, sender_name, subject, body):
+    with open('configuration/stanford/bin/email_params.txt', 'rt') as fin:
+        with open('email.txt', 'wt') as fout:
+            for line in fin:
+                line = line.replace('{RECIPIENT}', recipient).replace('{SENDER}', sender).replace('{SENDER_NAME}', sender_name).replace('{SUBJECT}', subject).replace('{BODY}', body)
+                fout.write(line)
+
+    fout = open('email.txt')
```

Comment:
This should be a `with` block:

```python
cmd = ['openssl', 's_client', '-crlf', '-quiet', '-connect', 'email-smtp.us-east-1.amazonaws.com:465']
with open('email.txt') as fout:
    call(cmd, stdin=fout)
call(['rm', 'email.txt'])
```

Reply:
@dcadams I made this change myself and pushed it to the branch;
I'm going to merge.

Reply:
or rather: any thing else before we merge?